app/jobs/send_purge_warnings_job.rb

```diff
@@ -9,7 +9,7 @@ def perform(*_args)
     not_sent = []
 
     # Get admin users
-    recipients = User.with_any_role(:admin)
+    recipients = User.with_any_role(:admin) | User.with_any_role(:super_user)
```

Comment:
Looking at this code, there actually was an existing bug here. `with_any_role` is a `Rolify` method which will be invalid once [this PR](https://github.com/SaraAlert/SaraAlert/pull/496) is merged. As this is now, I'm pretty sure it's been returning an empty array for `recipients` since we migrated away from Rolify. @holmesie for awareness - this may explain some Help Desk tickets we saw. 

This line should be changed to the following:
`recipients = User.can_receive_purge_warning?`

and you should add a new method to the `User` model in `user.rb`:
```
  def can_receive_purge_warning?
    (role?(Roles::ADMIN) || role?(Roles::SUPER_USER)) && jurisdiction&.name != 'USA'
  end
```

once you do that, you can remove the check for skipping USA admin's in the loop.